modules/ting/src/Search/TingSearchRequest.php

```diff
@@ -45,12 +45,75 @@ class TingSearchRequest {
 
   /**
    * @var string[]
-   *   Facets this query should query within. If empty all facets will be
-   *   queried.
+   *   Facets we want to query against. If specified the search-result should
+   *   enumerate facet-matches alongside the actual material matches.
    */
   protected $facets = [];
 
   /**
+   * @var int
+   *   Page offset. Defaults to 1 (first page)
+   */
+  protected $page = 1;
+
+  /**
+   * Sort directions.
+   *
+   * @var \Ting\Search\TingSearchSort[]
+   */
+  protected $sorts = [];
+
+  /**
+   * Raw provider-specific query string.
+   *
+   * @var string
+   */
+  protected $rawQuery;
```

Comment:
I am in general a bit uncertain what the "raw" part of the query actually means. Could that be explained better?

Reply:
I have documented the setter a bit more instead.